spec/support/controller_helpers.rb

```diff
@@ -1,14 +1,21 @@
-# some basic helpers to simulate devise controller methods in specs
```

Comment:
Hmm, to be honest, I wouldn't mind if there were more comments throughout the PR either above modules or functions similar to above